10.0/s2i/bin/run

```diff
@@ -1,2 +1,12 @@
 #!/bin/bash
-exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+
+run_wildfly() {
+  if [ "$DEV_MODE" == true ]; then
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0 -Dhot.deploy.enabled=true -DENABLE_JPDA=true
+  else
+    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+  fi
+}
+
+run_wildfly
```

Comment:
Following the node.js example

On Fri, Mar 18, 2016 at 5:58 PM, Ben Parees <notifications@github.com>
wrote:

> In 10.0/s2i/bin/run
> <https://github.com/openshift/sti-wildfly/pull/77#discussion_r56740322>:
>
> > @@ -1,2 +1,12 @@
> >  #!/bin/bash
> > -exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
> > +
> > +run_wildfly() {
> > +  if [ "$DEV_MODE" == true ]; then
> > +    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0 -Dhot.deploy.enabled=true -DENABLE_JPDA=true
> > +  else
> > +    exec /wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
> > +  fi
> > +}
> > +
> > +run_wildfly
>
> you don't need to wrapper this into a function
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/sti-wildfly/pull/77/files/797ce887ff800d24efb570955725033db9bf01e1#r56740322>
>